### ISSUE TITLE
Honor lang attribute

### ISSUE THREAD

**Issue by jsonn** [languagetool-org/languagetool-browser-addon#225]
When running a standalone server with the ngram tables, it is often handy to only install the dataset for the languages that are expected, i.e. German in my case. When the textarea contains i.e. "Test", it will be automatically detected as English and the standalone server is unhappy (error 500) about missing ngram files. This happens even if the document is tagged explicitly as German via `<html lang="de">` or similar means.

**Comment by danielnaber** [comment]
I don't think we should rely on the `lang` attribute - as the browsers also don't really use it that much, there a re probably many `lang` attributes with a wrong value.

**Comment by jsonn** [comment]
It is widely used by screen readers and other accessibility tools. It is normally also quite harmless if the default is bad, since the user can still override it. It just doesn't help when the server answers with a 500.
